service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/AbstractCallbackTest.java

```diff
@@ -71,16 +71,22 @@ protected AboutToStartOrSubmitCallbackResponse postAboutToStartEvent(CaseData ca
         return postAboutToStartEvent(asCaseDetails(caseData), SC_OK, userRoles);
     }
 
-    protected AboutToStartOrSubmitCallbackResponse postMidEvent(byte[] data, int expectedStatus) {
-        return postEvent(String.format("/callback/%s/mid-event", eventName), data, expectedStatus);
+    protected AboutToStartOrSubmitCallbackResponse postMidEventWithUserRole(CallbackRequest callbackRequest,
```

Comment:
Not sure this needed the name change, the var args means that you could call this method without roles

Reply:
for mid event there is overloaded

postMidEvent(callbackrequest, String additionalPath)

if i have 

postMidEvent(callbackrequest, String...roles)

then invocation like postMidEvent(callbackerror,"test") is ambiguous. So 2 options, 
have different method name or change param type to enum

Reply:
missed that, makes sense